### ISSUE TITLE
`neodoc` is archived / deprecated(?)

### ISSUE THREAD

**Issue by voxpelli** [LinusU/ts-readme-generator#6]
Just a heads up: I noticed that the neodoc repo has been archived: https://github.com/felixschl/neodoc

So while the npm module hasn't been marked as deprecated I'm pretty sure one can treat it as deprecated.

I myself currently use https://github.com/sindresorhus/meow for my CLI interfaces, though I only just discovered `docopt` when I looked at the code for this repo, so not sure if there's a newer / better / maintained docopt alternative for node.

**Comment by LinusU** [comment]
Yeah, I've know this for a while but still continue to use neodoc since I think it's such a good concept 😅 

Meow looks nice, there is also [`util.parseArgs`](https://nodejs.org/api/util.html#utilparseargsconfig) which is builtin to Node.js, but still experimental at the moment.
